services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -696,25 +653,14 @@ export class NoticeOfWorkApplication extends Component {
                 Edit Application Lat/Long
               </Menu.Item>
             </NOWActionWrapper>
-            <NOWActionWrapper permission={Permission.EDIT_PERMITS}>
-              <Menu.Item
-                key="edit-application-status"
-                className="custom-menu-item"
-                onClick={() => this.openUpdateStatusModal()}
-              >
-                Edit Application Status
-              </Menu.Item>
-            </NOWActionWrapper>
           </>
         )}
         {!isReview && Object.values(this.props.generatableApplicationDocuments).length > 0 && (
           <Menu.SubMenu key="generate-documents" title="Generate Documents">
             {Object.values(this.props.generatableApplicationDocuments)
               .filter(
                 ({ now_application_document_type_code }) =>
-                  now_application_document_type_code !== "PMA" &&
-                  now_application_document_type_code !== "PMT" &&
-                  now_application_document_type_code !== "NTR"
+                  now_application_document_type_code === "CAL"
```

Comment:
Can we do this logic by exclusion instead? So if the code doesn't equal rejection letter, withdrawal letter, etc. That way when we add new generatable documents we don't need to modify this condition.

Reply:
I updated it to be inclusive as this list seems to include less than it excludes - currently we're only showing the 'Acknowledge' letter here. 

The permit enclosed letter you just created will be attached to the issue permit modal, as is reject and withdrawal.

Reply:
I also included the permit enclosed letter here for now until it gets moved in a follow up PR